content/applications/finance/payment_providers.rst

```diff
@@ -355,6 +355,21 @@ Countries
 Restrict the use of the payment provider to a selection of countries. Leave this field blank to make
 the payment provider available in all countries.
 
+Currencies
+**********
+
+All payment providers have a different list of supported currencies. These lists are preconfigured
+and available in the :guilabel:`Configuration` tab. They serve as a first filter during payment
+operations, i.e. the payment provider is not displayed as available for payment if the currency is
+not in the supported list. As there might be errors, updates, and unknowns in these lists, it is
+possible to add or remove currencies.
+
+If the list of supported currencies is empty, it either means the list is too long to be displayed,
+or Odoo does not have information on that payment provider. The payment provider remains available,
+even though it is possible the payment will be refused at a later stage should the currency not be
+supported. This field is required if the user needs to select a currency when setting up their
+account.
```

Comment:
```suggestion
account.
```
Not sure why I didn't note on this earlier, perhaps I had more context then, but: which account are we talking about exactly ? I'm *guessing* the one from the provider, but as of right now, it is unclear.

Reply:
Yes, you guess right. For some providers (authorize & asiapay), the account is linked to a single currency that is defined when the account is created on the payment provider website.